utils/cliutils/commandsflags.go

```diff
@@ -531,6 +532,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  bundle,
 		Usage: "[Optional] If specified, only artifacts of the specified bundle are matched. The value format is bundle-name/bundle-version.` `",
 	},
+	gpgValidation: cli.StringFlag{
+		Name:  gpgValidation,
+		Usage: "[Optional] Path to the public GPG key file located on the file system.` `",
```

Comment:
Path to the public GPG key file located on the file system.
-->
Path to the public GPG key file located on the file system, used to validate downloaded release bundles.